### PR TITLE
Add regexp second test for vref filespecs

### LINKED ISSUE #1918: Is there regex in vref
Is there a way to include regex inside a vref, because that would be very useful in [this case](https://github.com/KSP-CKAN/NetKAN/pull/4779).

### PR DESCRIPTION
**Pull request by politas**
Closes #1918
Shouldn't impact any existing $vref filespecs, as they all have complete paths.

### CODE REVIEW

**Comment by politas** [comment]
#rebuild

File: Netkan/Services/ModuleService.cs [hunk 1]
```diff
@@ -116,7 +116,13 @@ private static AvcVersion GetInternalAvc(CkanModule module, ZipFile zipfile, str
...
                 for (var i = 0; i < files.Count; i++)
                 {
-                    if (files[i].Name == internalFilePath)
+                    Log.DebugFormat("Testing file '{0}' against path '{1}'", files[i].Name,internalFilePath);
+                    if (files[i].Name == internalFilePath) 
+                    {
+                        remoteIndex = i;
+                        break;
+                    }
+                    else if (Regex.IsMatch(files[i].Name, internalFilePath))
```
Comment (politas) [inline Netkan/Services/ModuleService.cs]: I thought about combining the tests, and it probably makes sense to do it that way. We're not cycling through the list a second time for the other test. 

Hang on, what's the "?" you've got there?

Comment (Olympic1) [inline Netkan/Services/ModuleService.cs]: the "?"  declares a nullable type, and means that the type before it may have a null value.

**Comment by politas** [comment]
The schema is for .ckan files, not .netkan files, so this doesn't affect it.

**Pull request merged**